CRM/Icaltimezone/Page/TimezoneICal.php

```diff
@@ -4,6 +4,10 @@
 class CRM_Icaltimezone_Page_TimezoneICal extends CRM_Core_Page {
 
   public function run() {
+    $gCalendar = CRM_Utils_Request::retrieveValue('gCalendar', 'Positive', 0);
+    if ($gCalendar) {
+      $this->assign('gCalendar', $gCalendar);
```

Comment:
@johntwyman my suggestion would be is to just do the assign no matter what its what Eileen has been doing for message templates

Reply:
Ok, so always pass it in and change from `{if isset()}` to `{if (gCalendar)}`